src/app/routes/base/base.component.ts

```diff
@@ -1,41 +1,40 @@
-import { Component } from '@angular/core';
-import { NavItem } from 'src/app/shared/nav-item';
+import { Component } from "@angular/core";
+import { NavItem } from "src/app/shared/nav-item";
 
 @Component({
-  selector: 'app-routes-root',
-  templateUrl: './base.component.html',
-  styleUrls: ['./base.component.css']
+  selector: "app-routes-root",
+  templateUrl: "./base.component.html",
+  styleUrls: ["./base.component.css"]
```

Comment:
Could we use single quotation marks instead? Should we potentially add `prettier` to this repo?

Reply:
Agreed, should be single quotes. Also the tslinter should catch that at compile time. It's a good one to have installed.